PyTDF/TDataFrame.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import print_function
+from .Node import Node
+
+class TDataFrame(Node):
+    
+    """
+    The Python equivalent
+    of ROOT C++'s TDataFrame 
+    data structure
+
+    """
+
+    def __init__(self, treename, filelist, npartitions=None):
+        # TODO (shravan97) : Implement different types of constructors
```

Comment:
We can start with this one, provided that filelist is actually a string (with possibly wildcards), as the real TDataFrame expects

Reply:
Actually this can also be sort of a pythonization of the constructor in C++ that expects a list of files, so we're good